tox.ini

```diff
@@ -9,14 +9,15 @@ deps = -r{toxinidir}/requirements.txt
 install_command = pip install {opts} {packages}
 
 [testenv:black]
-deps = black
+deps =
+  black >= 22.0, < 23.0
```

Comment:
I would suggest we use `>=23.0, <24.0`. I just checked with `23.1.0` and there are no problems.